### PR TITLE
chore: Improve Input Group documentation

### PR DESCRIPTION
**Pull request by salarenko**
#### Please provide a link to the associated issue.
Closes: #3199 

#### Please provide a brief summary of this pull request.
This PR:
- Fixes example breaking the documentation (improper usage of `fd-form-control`)
- Add barrel import for the examples
- Changes to static string binding when possible
- Removes unused files (.scss)

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Review by JKMarkowski — APPROVED** [review]
LGTM

File: apps/docs/src/app/core/component-docs/input-group/examples/input-group-states-example.component.html [hunk 1]
```diff
@@ -2,65 +2,63 @@
     <label fd-form-label>Information</label>
     <fd-form-input-message-group>
         <fd-input-group
-            fd-form-control
-            [addOnText]="'Button'"
-            [button]="true"
-            [state]="'information'"
-            [placeholder]="'Placeholder'"
+            button="true"
```
Comment (dimamarksman) [inline apps/docs/src/app/core/component-docs/input-group/examples/input-group-states-example.component.html:5]: Probably should be `[button]="true"`

**Comment by dimamarksman** [comment]
@salarenko why you changed `[button]="true"` to `button="true"`?

**Comment by dimamarksman** [comment]
`button="false"` will lead to `component.button = 'false'` and show button anyway
Please correct me if I'm wrong.

**Pull request merged**